tools/testinfra/runner/src/main.rs

```diff
@@ -32,23 +35,35 @@ struct Options {
     #[structopt(long = "buck-test-info")]
     spec: PathBuf,
 
-    /// Lists all unit tests and exits without running them
-    #[structopt(long)]
+    /// Lists unit tests and exits without running them
+    #[structopt(long, short)]
     list: bool,
 
     /// Path to generated test report in JUnit XML format
     #[structopt(long = "xml")]
     report: Option<PathBuf>,
 
+    /// Connection string of the DB used in stateful test runs
+    #[structopt(long = "db")]
+    conn: Option<String>,
+
+    /// Commit SHA-1 used to update the test DB on stateful runs
```

Comment:
nit: iirc git is going to transition to sha256, so just remove reference to SHA-1 here, since it ultimately doesn't matter

`/// Commit hash used to update the test DB on stateful runs`